spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/ml/MLPipelineEventProcessor.scala

```diff
@@ -154,53 +159,16 @@ class MLPipelineEventProcessor(
         val modelDirEntity = internal.mlDirectoryToEntity(uri, directory)
         val modelEntity = internal.mlModelToEntity(model, modelDirEntity)
         val uid = model.uid
-        cachedObjects.put(s"${uid}_modelDirEntity", modelDirEntity)
-        cachedObjects.put(s"${uid}_modelEntity", modelEntity)
+        internal.cachedObjects.put(s"${uid}_modelDirEntity", modelDirEntity)
+        internal.cachedObjects.put(s"${uid}_modelEntity", modelEntity)
 
       case name if name.contains("TransformEvent") =>
         val modeF = e.getClass.getDeclaredField("model")
         modeF.setAccessible(true)
         val model = modeF.get(e).asInstanceOf[PipelineModel]
-
-        val datasetInputF = e.getClass.getDeclaredField("input")
-        datasetInputF.setAccessible(true)
-        val inputdataset = datasetInputF.get(e).asInstanceOf[Dataset[_]]
-
-        val datasetOutputF = e.getClass.getDeclaredField("output")
-        datasetOutputF.setAccessible(true)
-        val outputdataset = datasetOutputF.get(e).asInstanceOf[Dataset[_]]
         val uid = model.uid
 
-        if (cachedObjects.contains(s"${uid}_modelEntity")) {
-          val logicalplan = inputdataset.queryExecution.analyzed
-          val tableEntities2 = logicalplan.collectLeaves().map {
-            case l: LogicalRelation => l.relation match {
-              case r: FileRelation => r.inputFiles.map(external.pathToEntity).toSeq
-              case _ => Seq.empty
-            }
-          }
-
-          val name = outputdataset.hashCode().toString
-          val tableEntities3 = getTableEntities(name)
-
-          val modelEntity = cachedObjects(s"${uid}_modelEntity").asInstanceOf[AtlasEntity]
-          val modelDirEntity = cachedObjects(s"${uid}_modelDirEntity").asInstanceOf[AtlasEntity]
-
-          val transformEntity = internal.mlTransformProcessToEntity(
-            model,
-            modelEntity,
-            List(modelEntity, tableEntities2.head.head),
-            List(tableEntities3.head))
-
-          atlasClient.createEntities(Seq(modelDirEntity, modelEntity, transformEntity)
-            ++ tableEntities2.head ++ tableEntities3)
-
-          cachedObjects.remove(s"${uid}_modelEntity")
-          cachedObjects.remove(s"${uid}_modelDirEntity")
-          logInfo(s"Created transFormEntity ${transformEntity.getGuid}")
-        } else {
-          logInfo(s"Transform Entity is already created")
-        }
+        logInfo(s"Cache for TransformEvent " + uid)
```

Comment:
`logInfo("Cache for TransformEvent " + uid)` or `logInfo(s"Cache for TransformEvent $uid")`?